<launch>
   <param name="leash_length" value="1.25"/>
   <param name="leash" textfile="$(find dogsim)/models/leash.model" />
   <node name="spawn_leash" pkg="gazebo" type="spawn_model" args="-param leash -gazebo -model leash -x 0 -y 0 -z 0" respawn="false" output="screen" />
   <node pkg="dogsim" type="leash_visualizer" name="leash_visualizer" />
</launch>
